pages/linux/lastcomm.md

```diff
@@ -0,0 +1,20 @@
+# lastcomm
+
+> Show last commands executed (from acct - GNU Accounting utilities).
+> More information: <https://manpages.debian.org/stable/acct/lastcomm.1.en.html>.
+
+- Print informations about all of the commands in the acct (record file):
+
+`lastcomm`
+
+- Display commands executed by {{user}}:
```

Comment:
It is not common (at least I didn't find on a quick search) to use tokens on command descriptions on tldr pages.

I believe we can just remove the curly brackets from the descriptions here and in other descriptions of this page.

Reply:
done. Thanks for the feedback.